docs/Contributing.asciidoc

```diff
@@ -670,13 +670,13 @@ docker pull registry.opensuse.org/devel/openqa/ci/containers/base:latest
 
 ```
 .circleci/build_local_docker.sh # will create image based on content of dependnencies.txt and autoinst
-docker run -it --rm -v $(pwd):/opt/testing_area localtest bash -c 'eval "$(t/test_postgresql | grep TEST_PG=)" && PERL5LIB=lib prove -v t/ui/25*'
+docker run -it --rm -v $(pwd):/opt/testing_area localtest bash -c 'make test-with-database TESTS=t/ui/25*'
```

Comment:
Should this be `test` rather than `test-with-dayabase`? To my mind that's an internal target.

Reply:
Oh I wasn't aware that `make test` does that now... fixed =)